examples/torch/object_detection/configs/ssd512_vgg_voc_magnitude_sparsity_int8.json

```diff
@@ -56,5 +56,6 @@
         }
       }
     }
-  ]
+  ],
+  "export_via_onnx": true
```

Comment:
https://github.com/openvinotoolkit/nncf/pull/2152#discussion_r1370900477 (about "--export-model-path") 
Even so, `export_via_onnx` is still requited because some models can not be converted without onnx.